### DISCUSSION TITLE
2024-01-23 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#292] [Meetings]
### Date: 2024-01-23, 3:00 – 4:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
| 3:00 - 3:05 | Welcome & opening remarks | Eza Hafeza |
| 3:05 - 3:45 | UCUM Release Plan | Steven Wagers, Tim Briscoe|
| 3:45 - 3:55 | New and Open Proposals (as time permits) | Eza Hafeza, All |
|3:50 - 3:55 | Additional Topics | All
| 3:55 - 4:00 | Topics for Next Meeting | All
||CIPM||
||New Proposals||
| 4:00 | Adjourn | Eza Hafeza |

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
| Continue asynchronous discussion on conformance / validation, [UCUM validation #250](https://github.com/orgs/ucum-org/discussions/250) | 2023-11-28 | Post first release | All |
| Advance membership of Jeremy Lieth to next phase | 2023-07-25 | 2023-11-14 | April Lackey |
| Collect use cases / user stories that may help inform license discussions | 2023-07-25 | 2023-11-14 | Tim Briscoe et. al. |

***LINKS TO REFERENCE MATERIALS***
[UCUM Committee_Meeting Summary_2023-11-28_Final](https://docs.google.com/document/d/1egrJrVuAKLzML9HaMkesnuOGq1Dt_s4i/edit?usp=drive_link&ouid=110875911742539584915&rtpof=true&sd=true)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release. 

Thank you,

Best Regards,
The LOINC Team
